repo/config/init_ipfs.go

```diff
@@ -146,7 +148,7 @@ func InitIpfs(identity native.Identity, mobile bool, server bool) (*native.Confi
 			},
 			DisableBandwidthMetrics: mobile,
 			DisableNatPortMap:       server,
-			EnableRelayHop:          false,
+			EnableRelayHop:          server,
```

Comment:
Just double checking, we only want this enabled when run in 'server' mode?

Reply:
I think so... maybe on desktop but probably requires not being behind a NAT